### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by catlam**

### CODE REVIEW

File: src/components/organisms/Footer.js [hunk 1]
```diff
@@ -1,9 +1,88 @@
-import React from 'react'
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (KshitizSharma11) [inline src/components/organisms/Footer.js]: The project is built on tailwind CSS strictly. Please avoid using bootstrap at all in this codebase.

**Review by KshitizSharma11 — CHANGES_REQUESTED** [review]
Please adhere to the changes requested.
1. Remove all bootstrap classes and replace them with appropriate tailwind classes.
2. Also remove bootstrap dependency from package.json and package-lock.json.
3. Stick to atomic structure for the component make seperare Footer-items.js file inside atoms export the items needed for the footer and import them inside Footer and map over the items to render them.

File: src/components/atoms/Footer-item.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```
Comment (KshitizSharma11) [inline src/components/atoms/Footer-item.js]: this wasnt what was expected only Footer-Items have to be exported in the form of a simple array later to be used inside the Footer component inside organisms to have clean code easy to maintain debug and scale

**Review by KshitizSharma11 — CHANGES_REQUESTED** [review]
Please try to make code easy to maintain and update less heavy and friendly for other developers

File: src/components/atoms/Footer-item.js (hunk 2 above)
Comment (catlam) [reply src/components/atoms/Footer-item.js:1]: I have edited the FooterItem but I don't know if it is as you described?

**Review by KshitizSharma11 — APPROVED** [review]
Looks Good And As Per Expectation.
Good work.

**Pull request merged**